app/lib/pdf_filler/ny201_v_pdf.rb

```diff
@@ -0,0 +1,26 @@
+module PdfFiller
+  class Ny201VPdf
+    include PdfHelper
+
+    def source_pdf_name
+      "it201v_1223"
+    end
+
+    def nys_form_type
+      "040"
+    end
+
+    def tax_year
+      2023
+    end
```

Comment:
Nit: wondering if we can avoid having this declared here? I know there's also `MultiTenantService.statefile.current_tax_year`, which is used as a parameter to initialize the It201. However, I don't want to get stuck in a rabbit hole about this, and it's perfectly fine to update this later, so if it makes the most sense right now to keep this here I won't question it.

Reply:
Forgot all about that - switched over to using it instead